apps/api/src/domain/models/term/repositories/arango-term-query-repository.ts

```diff
@@ -169,6 +170,7 @@ export class ArangoTermQueryRepository implements ITermQueryRepository {
                 bindVars,
             })
             .catch((reason) => {
+                // IS THIS CORRECT?  TRANSLATE TERM?
```

Comment:
Fix this or remove the catch. If we await, we might not need the catch.

Reply:
Fixed message

Reply:
Thanks